### PR TITLE
blueberry spec

### PR DESCRIPTION
**Pull request by yulin-dong**

### CODE REVIEW

File: docs/specs/blueberry.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+- _Standard Blocks_ may contain multiple transactions of the following types:
+  - CreateChainTx
+  - CreateSubnetTx,
+  - ImportTx,
+  - ExportTx,
+  - AddValidatorTx,
+  - AddDelegatorTx,
+  - AddSubnetValidatorTx.
+- _Proposal Blocks_ may contain a single transaction of the following types:
```
Comment (abi87) [inline docs/specs/blueberry.md:74]: we can go for the singural here `of the following type`

File: docs/specs/blueberry.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+  - RewardValidatorTx,
+- _Options blocks_, i.e. _Commit Block_ and _Abort Block_ do not contain any transactions.
+
+Note that each block has an header containing:
+
+- ParentID
+- Height
+- Time
+
+So the two main differences with respect to Apricot are:
```
Comment (abi87) [inline docs/specs/blueberry.md:84]: There is a thrid difference missing: AdvanceTimeTx transactions is not accepted anymore (for reasons explained in section `Chain Time Update Mechanism`

**Review by abi87 — CHANGES_REQUESTED** [review]
Some nits, maybe coming from little inconsistencies of the avalanchego repo docs?

File: docs/specs/blueberry.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
+# Blueberry Changes
+
+This document specifies changes in Avalanche Blueberry (AvalanchGo v1.8.0) from Apricot (AvalanchGo v1.7.x).
```
[the remark this answers is not in the document]
Comment (yulin-dong) [reply docs/specs/blueberry.md:3]: Apricot is the code name for AvalancheGo v1.1.x and up. Let me clarify that.

File: docs/specs/blueberry.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+- _Options blocks_, i.e. _Commit Block_ and _Abort Block_ do not contain any transaction.
+
+Note that _Atomic Blocks_ are deprecated as of AvalancheGo v1.7.0. They used to contain `ImportTx` and `ExportTx` which are now included into Standard Blocks.
+
+Each block has an header containing:
+
+- ParentID
+- Height
+
+Note that Apricot block headers do not contain any block timestamp.
```
[the remark this answers is not in the document]
Comment (yulin-dong) [reply docs/specs/blueberry.md:35]: nah

**Pull request closed without merging**